### PR TITLE
Slider for homepage

### PR DESCRIPTION
**Pull request by daltonfury42**

### CODE REVIEW

File: simplq/src/components/common/slider/slider.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```
Comment (maaverik) [inline simplq/src/components/common/slider/slider.stories.jsx:1]: It should be Slider.stories, right? Capital S like in index.

Comment (daltonfury42) [reply simplq/src/components/common/slider/slider.stories.jsx]: Done

**Comment by maaverik** [comment]
Could you add these pictures at the bottom too? That way the page would be complete except for the sidebar.
Screenshot from 2020-07-26 11-45-23

**Comment by maaverik** [comment]
And it would be good if you can reduce the padding here as well.
Screenshot from 2020-07-26 12-26-07

**Comment by daltonfury42** [comment]
All in separate PRs.. The bottom component PR is on it's way. Paddings, text fonts and sizes, I want to take it as a separate one with @akashkbaburajan

**Pull request merged**